docs/user/cropping-and-transforming.md

```diff
@@ -147,7 +147,7 @@ reader = PdfReader("resources/side-by-side-subfig.pdf")
 page = reader.pages[0]
 
 # Scale
-page.scale(0.5)
+page.scale_by(0.5)
```

Comment:
Maybe it was confused with `Transformation().scale` where the one of the arguments is optional.